### PR TITLE
Edit index page

### PR DESCRIPTION
**Pull request by soupdragon99**
Some minor copy edits.

### CODE REVIEW

**Comment by soupdragon99** [comment]
Two sets of edits, cleaning up a couple of links displaying incorrectly and a minor copy edit for clarity.

File: source/index.html.md.erb [hunk 1]
```diff
@@ -67,11 +67,11 @@ decision or standard.
 We’d like to thank all contributors for their suggestions and challenges as we continue to build and develop this repository, and seek the best way to build and operate our services so we can deliver at scale.
 
 If you think you can contribute a change to this repository which reflects
-current practice at GDS, make a pull request [in GitHub](https://github.com/alphagov/gds-tech)
+current practice at GDS, make a pull request in [GitHub](https://github.com/alphagov/gds-tech)
 and we'll discuss it at the Tech Ops Forum meeting and in the
 [#tech-ops-forum Slack channel](https://govuk.slack.com/messages/tech-ops-forum/).
 
-Alternatively contact email us at (<a href="mailto:the-gds-way@digital.cabinet-office.gov.uk?subject=feedback">The GDS Way</a>)
+Alternatively contact us at <a href="mailto:the-gds-way@digital.cabinet-office.gov.uk?subject=feedback">The GDS Way</a>
```
Comment (philandstuff) [inline source/index.html.md.erb]: this is missing a trailing full stop:

> `The GDS Way</a>`

should be

> `The GDS Way</a>`.

Comment (soupdragon99) [inline source/index.html.md.erb]: Always the smallest things, no problem. I have amended this, thanks for the 2i

**Comment by soupdragon99** [comment]
Have updated as suggested by @philandstuff

**Pull request merged**